lib/features/auto-place/AutoPlaceUtil.js

```diff
@@ -1,20 +1,21 @@
 import {
   asTRBL,
-  getOrientation
+  getOrientation,
```

Comment:
What is the purpose of this refactoring? 

I see a lot of changes, and all explainatory comments are being removed.

Reply:
To make it configurable and to make it easier to understand.